docs/smart_scripts.md

```diff
@@ -2,23 +2,25 @@
 
 [<-Back-to:World](database-world.md)
 
+The `smart_scripts` table has 30 attributes. It serves to make scripts in SQL language. The important thing is always to analyze, who is the event that motivates the execution of an action and of course, who is the objective. As a recommendation, you can review the scripts that are already inside the table, to understand how it works. The advantage, for which several use this method, is that it is not required to compile, when adding records, with restarting the server, if it is well programmed, you can appreciate the changes. Another reason is the portability, but it depends on the point of view of the developer, the response you can get.
+
 **Table Structure**
 
 | Field                 | Type        | Attributes | Key | Null | Default | Extra | Comment       |
 |-----------------------|-------------|------------|-----|------|---------|-------|---------------|
-| [entryorguid][1]      | int(11)     | signed     | PRI | NO   | NULL    |       |               |
+| [entryorguid][1]      | int(11)     |            | PRI | NO   |         |       |               |
```

Comment:
signed.

Reply:
The problem of that, is that in the SQL it does not say anything if it is signed, in all the tables I am going to have the same problem. On the other hand, when it is unsigned, if it is clarified, it is as if it is understood, at least in SQL. In all the tables, you are going to have me to correct the same problem.

Reply:
It is either signed or unsigned always.

If it is noy specified as unsigned it will be signed

Reply:
It has already been modified.

Reply:
yeye nice =) Just explaining.

You are too fast, I can't even work before I see a blue thing on github :P